global.json

```diff
@@ -6,5 +6,9 @@
     "MSBuild.Sdk.Extras": "3.0.44",
     "Microsoft.Build.NoTargets": "3.7.0",
     "Microsoft.DotNet.Arcade.Sdk": "8.0.0-beta.24310.5"
+  },
+  "sdk": {
+    "version": "8.0.100",
+    "rollForward": "latestMinor"
```

Comment:
I think we can use 
usePreviews : false and wil do the same thing

Reply:
good call, and we never need to update.